app/controllers/staff/activity_forms_controller.rb

```diff
@@ -280,6 +294,38 @@ def fstc_applies
     params.require(:activity).permit(:fstc_applies).fetch("fstc_applies", nil)
   end
 
+  def policy_marker_gender
+    params.require(:activity).permit(:policy_marker_gender).fetch("policy_marker_gender", nil)
+  end
+
+  def policy_marker_climate_change_adaptation
+    params.require(:activity).permit(:policy_marker_climate_change_adaptation).fetch("policy_marker_climate_change_adaptation", nil)
+  end
+
+  def policy_marker_climate_change_mitigation
+    params.require(:activity).permit(:policy_marker_climate_change_mitigation).fetch("policy_marker_climate_change_mitigation", nil)
+  end
+
+  def policy_marker_biodiversity
+    params.require(:activity).permit(:policy_marker_biodiversity).fetch("policy_marker_biodiversity", nil)
+  end
+
+  def policy_marker_desertification
+    params.require(:activity).permit(:policy_marker_desertification).fetch("policy_marker_desertification", nil)
+  end
+
+  def policy_marker_disability
+    params.require(:activity).permit(:policy_marker_disability).fetch("policy_marker_disability", nil)
+  end
+
+  def policy_marker_disaster_risk_reduction
+    params.require(:activity).permit(:policy_marker_disaster_risk_reduction).fetch("policy_marker_disaster_risk_reduction", nil)
+  end
+
+  def policy_marker_nutrition
+    params.require(:activity).permit(:policy_marker_nutrition).fetch("policy_marker_nutrition", nil)
+  end
+
```

Comment:
Probably not something for now, but I am a bit concerned that this controller is getting a bit unweildy, see my comment on @leeky's PR here https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/731#discussion_r522834283

Reply:
I see what you mean. I had a similar thought, and agree with you that's probably worth having a look in the future to refactor this a bit